open_spiel/games/chess.cc

```diff
@@ -110,7 +109,7 @@ void ChessState::MaybeGenerateLegalActions() const {
   if (!cached_legal_actions_) {
     cached_legal_actions_ = std::vector<Action>();
     Board().GenerateLegalMoves([this](const Move& move) -> bool {
-      cached_legal_actions_->push_back(MoveToAction(move));
+      cached_legal_actions_->push_back(MoveToAction(move, kMaxBoardSize));
```

Comment:
I don't think we need to plumb the constant through the whole code. Can be dropped from all calls.